src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -207,6 +211,10 @@ public String ipcMode() {
     return ulimits;
   }
 
+  public Boolean oomKillDisable() { return oomKillDisable; }
```

Comment:
This line (and the other getters added) cause your build to fail for code style.
Change
```java
public Boolean oomKillDisable() { return oomKillDisable; }
```
to
```java
public Boolean oomKillDisable() {
  return oomKillDisable;
}
```